src/python/grpcio_tests/tests_aio/benchmark/server.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 The gRPC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import logging
+import unittest
+
+from grpc.experimental import aio
+from src.proto.grpc.testing import messages_pb2
+from src.proto.grpc.testing import benchmark_service_pb2_grpc
+
+
+class BenchmarkServer(benchmark_service_pb2_grpc.BenchmarkServiceServicer):
+
+    async def UnaryCall(self, request, context):
+        payload = messages_pb2.Payload(body=b'\0' * request.response_size)
+        return messages_pb2.SimpleResponse(payload=payload)
+
+
+async def _start_async_server():
+    server = aio.server()
+
+    port = server.add_insecure_port(('localhost:%s' % 50051).encode('ASCII'))
+    servicer = BenchmarkServer()
+    benchmark_service_pb2_grpc.add_BenchmarkServiceServicer_to_server(
+        servicer, server)
+
+    await server.start()
+    await server.wait_for_termination()
+
+
+def main():
+    aio.init_grpc_aio()
+    loop = asyncio.get_event_loop()
+    loop.create_task(_start_async_server())
+    loop.run_forever()
```

Comment:
is this already used in someplace? where can we see the results?

Reply:
This is not automated right now. I just have a bunch of manual scripts to generate the result.

The important part of the benchmark is the client-side, I had two version, one is multiprocessing Python client, and one is Golang client. They are not fitting in this directory, so I just checked in this one file.

Reply:
will be `Aio` added as part of the current and official gRPC benchmark results?

Reply:
Yes, I will add it later. To add it into our benchmark suite:
1. The Async API should support both unary and streaming RPC;
2. Can work with sync API in same process (the control plane is also in gRPC).